.github/workflows/test-linux.yml

```diff
@@ -0,0 +1,25 @@
+name: 'Test (linux)'
+
+on:
+  workflow_call:
+
+jobs:
+  test:
+    name: 'ApiDiff'
```

Comment:
From what I can tell, Jon, the original author, prefers the following casing: APIDiff.

Reply:
You're right. Will update all occurances in a follow-up PR.